array-methods.js

```diff
@@ -5,11 +5,23 @@ var dataset = require('./dataset.json');
   greater than 100000
   assign the resulting new array to `hundredThousandairs`
 */
-var hundredThousandairs = null;
+
+const dataArray = dataset.bankBalances;
+
+var hundredThousandairs = dataArray.filter(
+  function(element){
+    if ((element.amount) > 100000){
```

Comment:
You could just return the result of the if conditional since this array method just wants a true/false value returned.

Reply:
By convention the function declaration should be on the line above.